src/lib/genesis_constants/genesis_constants.ml

```diff
@@ -27,6 +28,24 @@ module Fork_constants = struct
 end
 
 module Constraint_constants = struct
+  module Inputs = struct
+    type t =
+      { scan_state_with_tps_goal : bool
+      ; scan_state_tps_goal_x10 : int option
+      ; block_window_duration : int
+      ; scan_state_transaction_capacity_log_2 : int option
+      ; supercharged_coinbase_factor : int
+      ; scan_state_work_delay : int
+      ; coinbase : string
+      ; account_creation_fee_int : string
```

Comment:
AFAIU, this just follows an existing format. Could we still add a comment of whil `_int` is represented by a `string`?